pkg_resources/__init__.py

```diff
@@ -3118,19 +3114,15 @@ def activate(self, path: list[str] | None = None, replace: bool = False):
 
     def egg_name(self):
         """Return what this distribution's standard .egg filename should be"""
-        filename = "%s-%s-py%s" % (
-            to_filename(self.project_name),
-            to_filename(self.version),
-            self.py_version or PY_MAJOR,
-        )
+        filename = f"{to_filename(self.project_name)}-{to_filename(self.version)}-py{self.py_version or PY_MAJOR}"
```

Comment:
One of those cases I'd argue might be cleaner as `format` call. Especially with the length of the line and every f string values.

Reply:
Perhaps add extra `name` and `version` variables?